README.md

```diff
@@ -39,6 +39,7 @@ shows which versions of each generator are inside the docker image.
 
 | Docker image           | uniffi-rs version     | uniffi-bindgen-cs version | uniffi-bindgen-go version | uniffi-bindgen-cpp version |
 |------------------------|-----------------------|---------------------------|---------------------------|----------------------------|
+| v0.25.0-2              | v0.3.0+0.25.0         | v0.7.0+v0.25.0            | v0.2.0+v0.25.0            | **v0.2.0+v0.25.0**         |
```

Comment:
```suggestion
| v0.25.0-2              | v0.3.0+0.25.0         | v0.7.0+v0.25.0            | v0.2.0+v0.25.0            | **v0.2.2+v0.25.0**         |
```

Reply:
or do you only write the first major version?

Reply:
oops :D